### DISCUSSION TITLE
Impossible to cancel Webradio

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#248] [Bug Reports]
![Capture d’écran 2021-03-13 à 10 06 40](https://user-images.githubusercontent.com/77485841/111025395-126f5380-83e4-11eb-839e-f40c84601652.jpg)
Stange, I imported the very well done add-on Radio France Webradios. But As I previously add manually certain web radios like FIP, I cannot delete these ones. The delete feature has disappear for these ones.

**Comment by rern** [comment]
- Go to: `/srv/http/data/webradios`
- Delete: `https:||icecast.radiofrance.fr|fip-hifi.aac?id=radiofrance`
